### DISCUSSION TITLE
Trouble importing from pocket

### DISCUSSION THREAD

**Discussion by gregeeh** [wallabag/wallabag#6721] [Q&A] [answered]
Hi all,

Installed Wallabag in a docker container on my OMV NAS and I'm trying to import my 400 Pocket entries.  The import fails with a Cloudflare Gateway Timeout Error each time, requiring me to restart the container to regain access.

I can only get around 300 items imported.

Any ideas really appreciated.

TIA

[Here is the Portainer Stack](https://pastebin.com/raw/9vKTLv8E)

**Comment by j0k3r** [comment]
Are you running the import from the UI or using the command line?

**Reply by gregeeh** [reply]
From the UI.

**Reply by j0k3r** [reply, marked as answer]
You should try using the CLI instead you might have less timeout: https://github.com/wallabag/docker#import-worker

**Reply by gregeeh** [reply]
Thank you, will give that a try.
